src/components/AboutSection/index.jsx

```diff
@@ -0,0 +1,37 @@
+import statsArray from "./stats";
+
+const About = () => {
+  const items = [
+    {
+      title: "About Gravity Team",
+      paragraph:
+        "At Gravity Team, we are on the mission to balance the supply and demand across crypto markets worldwide. We are a crypto native market maker founded by traders, developers, and innovators who are strong believers and supporters of the future of decentralization and digital assets.",
+    },
+  ];
+
+  return (
+    <section className="bg-[#141619] text-white text-center">
```

Comment:
bg-[#141619] text-white text-center h-screen p-8 sm:p-16
```suggestion
    <section className="bg-[#141619] text-white text-center">
```

Reply:
**Mobile view if `h-screen` :**
![image](https://github.com/tcet-opensource/practice-react/assets/111864432/e992a052-60fa-4206-8d2f-87d75e522405)

---

**Desktop view if `h-full` :**
![image](https://github.com/tcet-opensource/practice-react/assets/111864432/b4decc7b-3209-4167-80e7-75e5c01675d8)